src/components/Editor/Editor.css

```diff
@@ -240,7 +240,7 @@ debug-expression-error {
   display: none;
 }
 
-.highlight-line .CodeMirror-line {
+.debugger:not(.can-rewind) .highlight-line .CodeMirror-line {
```

Comment:
what's this for?

Reply:
It was a quick fix so that WebReplay demos would be smoother

Reply:
can we do this for dark theme too?